src/kinto_http/session.py

```diff
@@ -44,11 +44,14 @@ def create_session(server_url=None, auth=None, session=None, **kwargs):
         elif "bearer" in auth.lower():
             # eg, "Bearer ghruhgrwyhg"
             _type, token = auth.split(" ", 1)
-            auth = kinto_http.BearerTokenAuth(token, type=_type)
+            auth = kinto_http.TokenAuth(token, type=_type)
```

Comment:
I know it's technically the same, but for consistency I would have left `BearerTokenAuth` here